test/parser.js

```diff
@@ -0,0 +1,146 @@
+
+
+var fs = require('fs');
```

Comment:
We can use ```const``` instead of ```var``` for all our imports since, we don't intend to reassign them.